keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/EnvironmentController.kt

```diff
@@ -11,7 +11,7 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 @RequestMapping(path = ["/environments"])
 class EnvironmentController(
-  private val repository: KeelRepository
+  private val environmentService: EnvironmentService
 ) {
 
```

Comment:
Reusing this method `/environments/{application}` since I'm like 99% confident that it is not used.